thunder/policy.py

```diff
@@ -193,8 +190,5 @@ def get_lr(self) -> List[float]:
             for param_group, mapping in zip_equal(self.optimizer.param_groups, self.current_mapping)
         ]
 
-    def state_dict(self) -> Dict[str, Any]:
-        return super().state_dict()
-
     def load_state_dict(self, state_dict: Dict[str, Any]) -> None:
         super().load_state_dict(state_dict)
```

Comment:
what's this for?

Reply:
load_state_dict is abstract, needs to be overriden

Reply:
then why do you call super here? does it have some default logic?

Reply:
It does